### DISCUSSION TITLE
Android App: Add home-screen widgets for calendar and emails

### DISCUSSION THREAD

**Discussion by Coxcopi** [tutao/tutanota#6877] [Ideas]
It would be great to be able to add tuta widgets to the home screen similar to gmail or the google calendar to provide an overview of the mail inbox or the current week/an overview of the next entries for the calendar.

**Comment by HarriBuh** [comment]
If I used a calendar app, I would expect a widget for my home screen. This is a must, even if it somehow lowered my security.

**Comment by Tourma** [comment]
Adding my voice to this.  I just started using Tuta, and it's one of the things that I liked with Proton's calendar.
